yas3fs/__init__.py

```diff
@@ -972,7 +983,7 @@ def __init__(self, options):
             self.current_user_aws_principalId = None
             try:
                 iam = boto.connect_iam()
-                self.current_user_principalId = u'AWS:'+iam.get_user()['get_user_response']['get_user_result']['user']['user_id']
+                self.current_user_principalId = 'AWS:'+iam.get_user()['get_user_response']['get_user_result']['user']['user_id']
```

Comment:
Does this continue to run properly on Python 2.7? I'm not sure why the original specified a unicode literal; my hunch is this was important on 2.x.

Reply:
I tried all the same commands on python 2.7 and 3.6